velox/expression/EvalCtx.h

```diff
@@ -273,7 +273,11 @@ class EvalCtx {
     return execCtx_->getVector(type, size);
   }
 
+  // Return true if the vector was moved to the pool.
   bool releaseVector(VectorPtr& vector) {
+    if (!vector) {
```

Comment:
nit: do we need this as VectorPool::release() already takes care of this case?

Reply:
i saw it there, but made it more explicit here, are you concerned with the cost?